react-project/src/components/List/List.js

```diff
@@ -0,0 +1,22 @@
+import { React } from "react";
+import { useSelector } from "react-redux";
+import "./List.scss";
+
+const List = () => {
+  const searchResults = useSelector((state) => state.location.searchResults);
+
+  return (
+    <div className="list">
+      {searchResults.map((result, i) => {
+        return (
+          <div key={i} className="list__item">
+            <span className="list__item--heading">{result.address}</span>
```

Comment:
heading is not a state but a component.
list__item-heading or list__item__heading

Reply:
Changed it to BEM